python/lsst/cp/pipe/cpCombine.py

```diff
@@ -701,6 +738,9 @@ def calibStats(self, exp, calibrationType):
             bad = ((ampImage.mask.array & noGoodPixelsBit) > 0)
             key = f"LSST CALIB {calibrationType.upper()} {amp.getName()} BADPIX-NUM"
             metadata[key] = bad.sum()
+            self.log.warn(f"Found {metadata[key]} pixels with "
+                          f"mask plane {self.config.noGoodPixelsMask} "
+                          f"for amp {amp.getName()}.")
```

Comment:
Two things.  First, this must be `self.log.warning()` and not `self.log.warn()`.  Second, this should only trigger if `metadata[key] > 0` or else we'll get flooded with non-warnings.